README.md

```diff
@@ -89,17 +89,255 @@ Name of the application pool.
+
+#### `max_processes`
+
+Indicates the maximum number of worker processes that would be used for the application pool.
+
+A value of '1' indicates a maximum of a single worker process for the application pool. This would be the setting on a server that does not have NUMA nodes.
+
+A value of '2' or more indicates a Web garden that uses multiple worker processes for an application pool (if necessary).
+
+A value of '0' specifies that IIS runs the same number of worker processes as there are Non-Uniform Memory Access (NUMA) nodes. IIS identifies the number of NUMA nodes that are available on the hardware and starts the same number of worker processes. For example, if you have four NUMA nodes, it will use a maximum of four worker processes for that application pool. In this example, setting maxProcesses to a value of '0' or '4' would have the same result
+
+#### `pinging_enabled`
+
+Specifies whether pinging is enabled for the worker process. Valid options `true` or `false`.
+
+#### `ping_interval`
+
+Specifies the time between health-monitoring pings that the WWW service sends to a worker process
+
+#### `ping_response_time`
+
+Specifies the time that a worker process is given to respond to a health-monitoring ping.  After the time limit is exceeded, the WWW service terminates the worker process
+
+#### `set_profile_environment`
+
+When setProfileEnvironment is set to True, WAS creates an environment block to pass to CreateProcessAsUser when creating a worker process. This ensures that the environment is set based on the user profile for the new process. Valid options `true` or `false`
+
+#### `shutdown_time_limit`
+
+Specifies the time that the W3SVC service waits after it initiated a recycle. If the worker process does not shut down within the shutdownTimeLimit, it will be terminated by the W3SVC service
+
+#### `startup_time_limit`
+
+Specifies the time that IIS waits for an application pool to start. If the application pool does not startup within the startupTimeLimit, the worker process is terminated and the rapid-fail protection count is incremented
+
+#### `user_name`
+
+Specifies the identity under which the application pool runs when the `identity_type` is `SpecificUser`
+
+#### `password`
+
+Specifies the password associated with the `user_name` property. This property is only necessary when the value of `identity_type` is `:SpecificUser`.  Note: To avoid storing unencrypted password strings in configuration files, this uses AppCmd.exe. This encrypts the password automatically before it is written to the XML configuration files. This provides better password security than storing unencrypted passwords."
```

Comment:
if `:SpecificUser` is a string value, it should be in single quotes instead of backticks

Also, this sentence has a rogue quotation mark at the end.